tests/cpp/Field/fieldlistview_tests.cc

```diff
@@ -103,131 +108,145 @@ GPU_TYPED_TEST_P(FieldListViewTypedTest, BasicCapture) {
 
 // TODO: Add test for having multiple FL contain the same field
 
+//------------------------------------------------------------------------------
+// multi-scope
+//------------------------------------------------------------------------------
 GPU_TYPED_TEST_P(FieldListViewTypedTest, MultiScopeAndTouch) {
 
   const double val = 4.;
   NodeList_t nl1 = gpu_this->createNodeList("NL1");
   NodeList_t nl2 = gpu_this->createNodeList("NL2");
   {
-    FieldDouble field("TestField1", nl1, val);
+    FieldDouble field1("TestField1", nl1, val);
     FieldDouble field2("TestField2", nl2, val);
-    FieldListDouble field_list;
+    field1.setCallback(gpu_this->f_callback());
+    field2.setCallback(gpu_this->f_callback());
 
-    field_list.appendField(field);
+    FieldListDouble field_list;
+    field_list.appendField(field1);
     field_list.appendField(field2);
 
     const size_t numFields = field_list.size() ;
 
     { // Scope 1
-    auto fl_v = field_list.toView(gpu_this->fl_callback(), gpu_this->f_callback());
+      auto fl_v = field_list.view(gpu_this->fl_callback());
 
-    DEBUG_LOG << "Start Kernel 1";
-    RAJA::forall<TypeParam>(TRS_UINT(0, numFields),
-      [=] SPHERAL_HOST_DEVICE (size_t i) {
-        SPHERAL_ASSERT_EQ(fl_v.size(), numFields);
-        SPHERAL_ASSERT_EQ(fl_v[i].size(), N);
-      });
-    DEBUG_LOG << "Stop Kernel 1";
+      DEBUG_LOG << "Start Kernel 1";
+      RAJA::forall<TypeParam>(TRS_UINT(0, numFields),
+                              [=] SPHERAL_HOST_DEVICE (size_t i) {
+                                SPHERAL_ASSERT_EQ(fl_v.size(), numFields);
```

Comment:
We should decide what sort of spacing we want for these RAJA loops.

Reply:
Yep, in this case this was what happened automatically with emacs formatting.  That opening parenthesis pushes it way to the right.